src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -193,7 +199,7 @@ export const Sidebar = React.forwardRef<HTMLDivElement, Props>(
           refetchAll(),
         )
       }
-    }, [])
+    }, [redeemCode, redeemedCampaigns, refetchAll])
```

Comment:
won't this retrigger once we change the redeemed campaigns now? Meaning we might get some kind of infinite loop or?

Reply:
No infinite loops when I tested it but maybe I'm missing something? Maybe there's something I don't know about `redeemedCampaigns`?

Reply:
Looked a bit closer at it now and yes, it could have caused an infinite loop if I hadn't wrapped it in `useMemo` like the nice linter suggested to me :)